src/main/java/lotto/domain/Lotto.java

```diff
@@ -6,28 +6,41 @@
 import java.util.stream.IntStream;
 
 public class Lotto {
-    private List<Integer> rangeNumbers = IntStream.rangeClosed(1, 45).boxed().collect(Collectors.toList());
+    private final static int START_NUMBER = 1;
+    private final static int END_NUMBER = 45;
+    private final static int NUMBER_OF_LOTTO_NUMBERS = 6;
+    private final static int LOTTO_PRICE = 1000;
+
+    private static List<Integer> lottoNumbers = IntStream.rangeClosed(START_NUMBER, END_NUMBER).boxed().collect(Collectors.toList());
     private final List<Integer> lotto;
 
 
     public Lotto() {
-        Collections.shuffle(rangeNumbers);
-        lotto = rangeNumbers.stream()
-                .limit(6)
-                .sorted()
-                .collect(Collectors.toList());
+        this(randomNumberGenerate());
     }
 
     public Lotto(List<Integer> randomNumbers) {
-        lotto = randomNumbers;
+        this.lotto = randomNumbers;
     }
 
     public Rank winningResult(WinningNumber winningNumber) {
-        int matchCount = (int) winningNumber.getWinningNumbers()
-                .stream()
-                .filter(number -> this.lotto.contains(number))
+        int matchCount = (int) winningNumber.stream()
+                .filter(this.lotto::contains)
                 .count();
-        return Rank.valueOf(matchCount);
+        Boolean isBonusMatch = this.lottoNumbers.contains(winningNumber.getBonusNumber());
```

Comment:
참조타입을 쓰신이유가 있나요? 별다른 이유가 없다면 원시타입으로 제공하는게 좋습니다. 🙂 
```suggestion
        boolean isBonusMatch = this.lottoNumbers.contains(winningNumber.getBonusNumber());
```
[Java에서 원시타입 vs 참조타입 어떤 걸 사용해야 할까?](https://siyoon210.tistory.com/139)